.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- In the following, please describe your issue in detail! -->
+<!-- If some of the sections do not apply, just remove them. -->
+
+### Short description
+<!-- This should summarize the issue. -->
+
+### Code to reproduce
+```python
+import pyqtgraph as pg
+import numpy as np
+# This is a placeholder for a minimal working example that reproduces the issue.
+```
+
+### Expected behavior
+<!-- What should happen? -->
+
+### Real behavior
+<!-- What happens? -->
+
+```
+An error occurred?
+Post it inside these 'code fences'!
+```
+
+### Tested environment(s)
```

Comment:
One more bullet point I'd add is installation method (pip, conda, system packages, etc.)